### PR TITLE
[13.0][MIG] Module stock_picking_invoicing

### PR DESCRIPTION
**Pull request by mbcosta**
Solved issues of PR migration module stock_picking_invoicing to version 13 https://github.com/OCA/account-invoicing/pull/836:

* Cherry-Picking of PR https://github.com/OCA/account-invoicing/pull/969
* Included the Action 'Create Draft Invoices'
image

image

cc @rvalyi @renatonlima @sergioeix

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]
code review + tests. Thank you @mbcosta for porting the last 12.0 bits.

**Comment by mbcosta** [comment]
Removed method _prepare_refund, by description it was necessary in v12 for "Inherit to put link picking of the invoice into the new refund", but seems unnecessary anymore:

image

image

**Review by marcelsavegnago — APPROVED** [review]

**Comment by Rad0van** [comment]
Nice. I couldn't find time to finish my attempt. Great you did it properly.

File: stock_picking_invoicing/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Leonardo Pistone <leonardo.pistone@agilebg.com>
+* Daniel Sadamo <daniel.sadamo@kmee.com.br>
+* Alex Comba <alex.comba@agilebg.com>
+* Florent THOMAS <florent.thomas@mind-and-go.com>
+* François Honoré <francois.honore@acsone.eu>
+* Magno Costa <magno.costa@akretion.com.br>
+* Luis Felipe Mileo <mileo@kmee.com.br>
+* Radovan Skolnik <radovan@skolnik.info>
```
Comment (Rad0van) [inline stock_picking_invoicing/readme/CONTRIBUTORS.rst:9]: @mbcosta not sure I deserve to be left here as I didn't finish the migration.

Comment (rvalyi) [reply stock_picking_invoicing/readme/CONTRIBUTORS.rst:9]: let's say now you have some pressure to become an OCA hero with your future contributions ;-)

**Comment by rvalyi** [comment]
IMPORTANT NOTE: normally the 1st 13.0 version number should have been `13.0.1.0.0` as per the OCA spec https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#tasks-to-do-in-the-migration
Now given that this migration took one year with several 12.0 front-ports and new sub-versions during the 13.0 port let's keep the current number to help if somebody has been using the PR branch and wants to upgrade. Changing it now would force us to introduce some strange commit or erase the front-ports details, so this wouldn't be very nice. But let's keep the OCA spec in mind for future migrations. cc @mbcosta @renatonlima @marcelsavegnago @Rad0van @sergioeix

File: stock_picking_invoicing/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (Rad0van) [reply stock_picking_invoicing/readme/CONTRIBUTORS.rst:9]: > let's say now you have some pressure to become an OCA hero with your future contributions ;-)

I'll do my best...

Comment (mbcosta) [reply stock_picking_invoicing/readme/CONTRIBUTORS.rst:9]: Hi @Rad0van , I don't see problems if you want to be a contributor. For a while we thought this module was just used in Brazil, is better have people from other countries using, testing and giving feedback. If you have problems with the module and need to create an Issue or PR you can mark me.

**Comment by mbcosta** [comment]
Thanks @rvalyi @marcelsavegnago @Rad0van  for review.

**Pull request merged**